tox.ini

```diff
@@ -12,10 +12,13 @@ python =
 
 [testenv]
 deps =
-    pytest
-    -r requirements.txt
+  .[test]
```

Comment:
This also installs pyargus.

Reply:
Ouch... I didn't think about that. Guess I'll break it out into `tests/requirements.txt` instead.